Source/WTF/wtf/HashTable.h

```diff
@@ -823,13 +817,13 @@ DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(HashTable);
     ALWAYS_INLINE void HashTable<Key, Value, Extractor, HashFunctions, Traits, KeyTraits, Malloc>::addUniqueForInitialization(T&& key, NOESCAPE const std::invocable<> auto& functor)
     {
         ASSERT(m_table);
-        checkHashTableKey<Key, Value, Extractor, HashFunctions, Traits, KeyTraits, HashTranslator, ShouldValidateKey::No>(key);
         invalidateIterators(this);
 
         internalCheckTableConsistency();
 
         Value* entry = lookupForReinsert<HashTranslator>(key);
         HashTranslator::translate(*entry, std::forward<T>(key), functor);
+        validateKey<ShouldValidateKey::No>(*entry);
```

Comment:
I think it would be clearer to have a non-template function to use in a case like this. It’s so wordy to pass the template argument.